### ISSUE TITLE
dont work diy saroo

### ISSUE THREAD

**Issue by xdemax** [tpunix/SAROO#184]
hello! i build two saroo v1.3 -- number one working without problem, number two view only 15 colored square after boot (without string  'SRAOOO start!'). i check all 164245 and ram chips with saroo number one board -- its work. resoldering fpga/mcu, but i see only 15 colored square. have any idea?

**Comment by 07saga** [comment]
FPGA virtual soldering or damage

**Comment by moneytac1** [comment]
Did you figure anything out?

**Comment by xdemax** [comment]
yes. its bad ram -- most likely, when removing the memory from the working board, it failed. i install new ram and saroo work perfect.
